### DISCUSSION TITLE
is there treeview in customtkinter?

### DISCUSSION THREAD

**Discussion by Ryzelz** [TomSchimansky/CustomTkinter#1575] [Q&A] [answered]
as im working, there's no treeview. any ideas what should I do?

![image](https://user-images.githubusercontent.com/100330023/236652654-5cabf8f5-8197-4c97-b2e2-7a164d51c748.png)

**Comment by Akascape** [comment, marked as answer]
@Ryzelz 
These may help:

https://github.com/TomSchimansky/CustomTkinter/discussions/524#discussioncomment-3898307

https://github.com/TomSchimansky/CustomTkinter/discussions/1483
